helm/examples/azure/sa/create-sa.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+WORK_DIR=$(dirname $0)
+source ${WORK_DIR}/env-sa
+
+if [[ -z "$SA_RESOURCE_GROUP" ]] || [[ -z "$SA_LOCATION" ]] || [[ -z "$STORAGE_ACCOUNT_NAME" ]]
+then
+   echo "Miss one of SA_RESOURCE_GROUP, SA_LOCATION and STORAGE_ACCOUNT_NAME"
+   exit 1
+fi
+
+[[ -n "$SUBSCRIPTION" ]] && az account set --subscription $SUBSCRIPTION
+
+VALUES_FILE=${WORK_DIR}/../hpcc-azurefile/values.yaml
+SECRET_NAME=$(cat $VALUES_FILE | grep "^[[:space:]]*secretName:"|cut -d':' -f2|sed 's/[[:space:]]//g')
+SECRET_NAMESPACE=$(cat $VALUES_FILE | grep "^[[:space:]]*secretNamespace:"|cut -d':' -f2|sed 's/[[:space:]]//g')
+SHARE_NAME=$(cat $VALUES_FILE | grep "^[[:space:]]*shareName:"|cut -d':' -f2|sed 's/[[:space:]]//g')
```

Comment:
This won't work in general. You can't rely on the settings in this single values.yaml being where the settings finally come from. i.e. the settings may come from --set specifiers when install hpcc-azurefile, or from any passed in alternative yaml files. e.g.
```
helm install examples/azure/hpcc-azurefile --set secretName=myscret --set shareName=myshare
```

is going to break the above.
A user of these scripts should provide these settings to create-sa.sh (via command line preferably),
and *then* supply them to the hpcc-azurefile chart however they wish to, whether via their own custom values yaml or override yaml, or via --set 's.

Reply:
I will allow to set these in env-sa as well then.